oak_attestation_verification/src/verifier.rs

```diff
@@ -49,28 +67,12 @@ pub fn verify(
     endorsements: &Endorsements,
     reference_values: &ReferenceValues,
 ) -> AttestationResults {
```

Comment:
Pre-existing: I think this function should return normal Rust types, including the adctual strongly typed keys (rather than serialised keys in a proto). We could then implement the `From` trait to convert between the strongly typed Rust result struct and the proto if we need to proto for FFI (to support a C wrapper).